crates/containerd-shim-wasm/doc/doc.md

```diff
@@ -1,36 +0,0 @@
-A library to help build containerd shims for wasm workloads.
-
-## Usage
```

Comment:
Why is the usage doc being removed?

Reply:
This is `Usage` is duplicated in `README.md`.
It was basically `cat doc/header.md doc/doc.md > README.md`
This PR proposes not using `doc/*.md`, and just work on `README.md` directly.